### PR TITLE
added unit tests to footer and aquipollutionate

### PR DESCRIPTION
**Pull request by sadiq-b**
wrote some unit tests for the footer and aquipollutionate components
Screenshot (843)

### CODE REVIEW

**Comment by shehabadel** [comment]
There are some conflicts that need to be resolved @sadiq-b

**Review by SanjaySinghRajpoot — APPROVED** [review]
Changes look good to me. Just have a look at the merge conflicts!!

File: src/App.js [hunk 1]
```diff
@@ -346,6 +346,10 @@ function App() {
   } else {
     return (
       <div className="fade">
+<<<<<<< HEAD
```
Comment (shehabadel) [inline src/App.js:349]: Remove this line

File: src/App.js [hunk 2]
```diff
@@ -346,6 +346,10 @@ function App() {
...
+=======
```
Comment (shehabadel) [inline src/App.js:350]: Remove this line as well

File: src/App.js [hunk 3]
```diff
@@ -346,6 +346,10 @@ function App() {
...
+        <Header />
+>>>>>>> main
```
Comment (shehabadel) [inline src/App.js:352]: Remove this line finally then save the file.

Comment (sadiq-b) [reply src/App.js]: thanks, I have made the changes kindly check my new commit for the latest updates @shehabadel

**Comment by shehabadel** [comment]
Looking good I'll merge now @sadiq-b @SanjaySinghRajpoot @somadisingh

**Pull request merged**